HKMP/Game/Client/ClientManager.cs

```diff
@@ -602,10 +604,17 @@ ModSettings modSettings
                 return;
             }
 
-            // Destroy corresponding player
-            _playerManager.DestroyPlayer(id);
+            // Recycle corresponding player
+            _playerManager.RecyclePlayer(id);
 
             playerData.IsInLocalScene = false;
+            foreach (Transform child in playerData.PlayerObject.transform)
+            {
```

Comment:
Code-style inconsistency.
```suggestion
            foreach (Transform child in playerData.PlayerObject.transform) {
```